docs/cli.md

```diff
@@ -138,6 +138,20 @@ and forcing the scripts and sources to be placed in `/opt` directory:
 $ sti build --scripts-url=file://stiscripts --destination=/opt git://github.com/mfojtik/sinatra-app-example openshift/ruby-20-centos7 ruby-app
 ```
 
+# sti rebuild
+
+The `sti rebuild` command is used to rebuild the image already build using S2I,
+or the image that contains the required S2I labels.
```

Comment:
@soltysh yes, I want to add them to openshift docs and link to that... as follow up